rir/src/compiler/translations/rir_2_pir/rir_2_pir.cpp

```diff
@@ -525,22 +525,40 @@ bool Rir2Pir::compileBC(const BC& bc, Opcode* pos, Opcode* nextPos,
         break;
     }
 
-    case Opcode::subassign1_: {
+    case Opcode::subassign1_1_: {
         Value* idx = pop();
         Value* vec = pop();
         Value* val = pop();
         push(insert(new Subassign1_1D(val, vec, idx, env, srcIdx)));
         break;
     }
 
-    case Opcode::subassign2_: {
+    case Opcode::subassign2_1_: {
         Value* idx = pop();
         Value* vec = pop();
         Value* val = pop();
         push(insert(new Subassign2_1D(val, vec, idx, env, srcIdx)));
         break;
     }
 
+    case Opcode::subassign1_2_: {
+        Value* idx1 = pop();
+        Value* idx2 = pop();
```

Comment:
Aren't these swapped? And for the other opcode as well
```
Value* idx2 = pop();
Value* idx1 = pop();
```

Reply:
Or, seeing travis is green, how come this order is ok? :)

Reply:
They should be same order as `extract*_2_` so maybe they are swapped. Travis is probably green because PIR doesn't do anything with `Subassign*_2_` (which is also why `rhs`/`lhsValue` is OK, I'll fix both)